changelog/8.7.1.yaml

```diff
@@ -0,0 +1,111 @@
+version: 8.7.1
```

Comment:
Belated comment, looks like we missed https://github.com/elastic/elastic-agent/commit/2d90292e4d1843e7dd2417ea42804f1788d77c7e here and in the official release notes

Reply:
Indeed, we did.  Thanks for catching.  It looks like [the PR for that change](https://github.com/elastic/elastic-agent/pull/2448/files) didn't add a CHANGELOG fragment file, which is why it didn't get included here.  I'll make a follow up PR to directly add an entry to the `8.7.1.yaml` file.

Reply:
> I'll make a follow up PR to directly add an entry to the `8.7.1.yaml` file.

https://github.com/elastic/elastic-agent/pull/2577

Reply:
cc: @kilfoyle, for reviewing https://github.com/elastic/elastic-agent/pull/2577 and for incorporating the missed entry into the official release notes as well.